js/ui/list.js

```diff
@@ -1,10 +1,12 @@
-import ListEdit from './list/ui.list.edit.search';
-import registerComponent from '../core/component_registrator';
-// STYLE list
-
-registerComponent('dxList', ListEdit);
+import List from './list_light';
```

Comment:
I don't want to put it to the `list` folder. All API modules are located in the 'ui' and folders are for internal modules.
The name is still discussable. Maybe `list_base` or `list_core`.

Reply:
This is up to you but: for the public API it may be like
`devextreme/ui/list` <--contains everything
`devextreme/ui/list/core` <--logically located inside the list, but only the core part (this is the `list_light` in this PR). So a user can understand that this is a sub-part of a list
`devextreme/ui/list/modules/selection` <-- modules. We can even move it to the outer scope like `devextreme/ui/list/selection` (if the list and other widgets do not contain anything inside)